bundles/framework/oskariui/resources/locale/en.js

```diff
@@ -2,6 +2,117 @@ Oskari.registerLocalization({
     lang: 'en',
     key: 'oskariui',
     value: {
-        otherLanguages: 'Other languages'
+        otherLanguages: 'Other languages',
+        "VisualizationForm": {
+            "subheaders" : {
+                styleFormat: 'Style format',
+                name: "Style name",
+                style: "Style"
+            },
+            "tooltips": {
+                "dot": "Edit point feature style",
+                "area": "Edit area feature style",
+                "line": "Edit line feature style"
+            },
+            "validation": {
+                "title": "The given data are invalid:",
+                "lineSize": "The line width is invalid. It must be between 1 and 50.",
+                "borderWidth": "The line width is invalid. It must be between 1 and 50."
```

Comment:
Since all the fields now seem to have an object with key label, would it make sense to have that fields validation message beside that label instead of this common validation key?

Reply:
I'll add validation messages to localisation when validations are done.